src/tokens.spec.ts

```diff
@@ -1,12 +1,12 @@
-import { Token, TokenComputer } from "./tokens";
 import { assert } from "chai";
+import { Token, TokenComputer } from "./tokens";
 
 describe("test token computer", async () => {
     const tokenComputer = new TokenComputer();
 
     it("should test if token is fungible", async () => {
-        const fungibleToken = new Token("TEST-123456", 0);
-        const nonFungibleToken = new Token("NFT-987654", 7);
+        const fungibleToken = new Token("TEST-123456", 0n);
```

Comment:
I'd say it's more explicit to call `BigInt()` than to add `n` at the end of the value, but it's easier and shorter this way.

Reply:
As discussed separately, we can recommend usage of bigint literal (*n).